packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -1,17 +1,21 @@
-"use strict";
```

Comment:
Do we need to rename it as `build-dist.mjs`?

Reply:
I added a `package.json` file with `"type": "module"`, similarly to what we do in the other `scripts` folders.